processing/src/main/java/io/druid/query/groupby/GroupByQuery.java

```diff
@@ -572,6 +573,10 @@ private static void verifyOutputNames(
         throw new IAE("Duplicate output name[%s]", postAggregator.getName());
       }
     }
+
+    if (outputNames.contains(Column.TIME_COLUMN_NAME)) {
+      throw new IAE("'__time' cannot be used as an output name for dimensions, aggregators, or post-aggregators.");
```

Comment:
Might as well do `%s` and `Column.TIME_COLUMN_NAME` here since you went to the trouble of doing it in the if check.